xunit-autolabeler-v2/ast_parser/python/source_parsers/flask_router_test.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import ast
+import os
+import pytest
+import unittest
+
+
+from . import flask_router
+
+
+def _create_fixture(example_path):
```

Comment:
@tmatsuo @martinvarelaj there's gotta be a better way to do this - any ideas, since I'm a bit rusty with Pytest? 🙂

Reply:
I was hoping y'all would know of something off the top of your head. 🙂 

Being more explicit - what I want to do here is a _class-level parametrized fixture_. Doing one or the other (class-level *or* parameterized) seems doable, but I'm not sure if you can combine them. (From my understanding, class-level fixtures [require the use of `pytest.mark.fixture()`](https://docs.pytest.org/en/3.0.1/unittest.html), which [doesn't support parametrized arguments](https://docs.pytest.org/en/stable/fixture.html#usefixtures).

**TL;DR:** ...maybe there _isn't_ a better way.